coverage_config_aarch64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 76.7,
+  "coverage_score": 77.0,
```

Comment:
Unrelated to this PR, but you might want to also update the rust-vmm-ci to the latest version that essentially disables this test (as it is very unreliable).